sdf.md

```diff
@@ -1176,8 +1176,9 @@ In SDF 1.0, a similar concept was called `subtype`.
 
 Data can be a choice of named alternatives, called `sdfChoice`.
 Each alternative is identified by a name (string, key in the JSON
-object used to represent the choice) and a set of dataqualities
-(object, the value in the JSON object used to represent the choice).
+map used to represent the choice) and a set of dataqualities
+(inner JSON map, the value used to represent the
+choice in the outer JSON map).
```

Comment:
This sentence has become a bit difficult to read due to the diverge of the terms used for the outer JSON map – is there maybe a way to simplify it?

Reply:
Instead of simplifying it, I clarified it (see below)...